apiary.apib

```diff
@@ -71,6 +71,18 @@ Any response that does not provide a ```200 OK``` response code should be treate
 
     + Attributes (Heartbeat)
 
+# Group Account
+
+## Account Collection [/accounts/{accountId}]
+
++ Parameters
+  + accountId: 1234567890 (number, required) - The Dude Platform account the user belongs too.
```

Comment:
This also needs ```#INT``` in the description so it forces the "int" type when the SDK is generated